web-server/helpers.js

```diff
@@ -167,3 +170,7 @@ module.exports = app => {
   })
 
 }
+
+function getTitleFromHTML(content) {
+  return content.match(/>(.*)</)[1]
```

Comment:
this is sneaky. How does this work?

Reply:
`content` is a string of HTML which is the result of parsing markdown.
Line 175 looks in this string for the contents of the first html tag which is the title.
We'll update it to look for the first `h1` tag to make it more robust.